src/pydap/net.py

```diff
@@ -29,44 +31,39 @@ def raise_for_status(response):
         )
 
 
-def follow_redirect(url, application=None, session=None,
-                    cookies_dict=None):
+def follow_redirect(url, application=None, session=None):
     """
     This function essentially performs the following command:
     >>> Request.blank(url).get_response(application)
 
     It however makes sure that the request possesses the same cookies and
     headers as the passed session.
-
-    It recursively follows 302 redirects.
```

Comment:
In fact this change was made to make future development easier. Here's the thinking:
1. ``get_cookies`` set the proper cookies in ``session``. This is checked using the ``check_url`` option, so we know it works.
2. ``requests`` is so well maintained that there are no reason why querying the url should fail: after all, we have checked that the credentials work.
3. ``requests`` handles all the cookies and the redirects. We simply have to use its result and pass it to ``webob`` for the last mile.

Makes sense? I can document that in the code if it does.